stock_picking_batch_split/wizards/stock_picking_batch_split_wizard.py

```diff
@@ -0,0 +1,15 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3.0 (http://www.gnu.org/licenses/agpl-3.0.html)
+
+from odoo import models, fields
+
+class StockPickingBatchSplit(models.TransientModel):
+    _name="stock.picking.batch.split.wizard"
+    _description="Split picking"
+
+    product_id = fields.Many2one("product.product", required=True)
```

Comment:
If its required we cant open wizard
```suggestion
    product_id = fields.Many2one("product.product")
```